docs/Contributing-back.rst

```diff
@@ -0,0 +1,496 @@
+
+.. note:: Please use a concise commit message that describes the changes you made.
+
+To create a single commit for *all* the changes that were made, use ``git commit -am "<message"``.
+
+For example:
+
+.. code:: shell
+
+  # first, check changes to make sure we want to commit them all at once
+  git status
+  git diff
+
+  git commit -am "example commit message for all the changes that were made"
+
+To verify that your work was committed, use ``git log`` to see all commits
+on the current branch. Use ``git log --stat`` and/or ``git log --diff`` to see
+more details about which changes are included in each of the commits.
+
+
+.. _contributing_back_creating_pull_requests_push:
+
+Pushing your branch
++++++++++++++++++++
+
+Once you have committed your changes to a branch, you should *push* your
+branch to your fork of the GitHub repository using ``git push``.
+
+For example, to push the ``mybranch`` branch to the ``myfork`` remote GitHub repository:
+
+.. code:: shell
+
+  git push myfork mybranch
+
+Note that this is equivalent to making your work public.
+
+
+.. _contributing_back_creating_pull_requests_open_pr:
+
+Opening the pull request
+++++++++++++++++++++++++
+
+To open a pull request using the branch you pushed,
+you should use the GitHub interface, and follow the steps outlined below.
+
+.. note:: Replace '``EXAMPLE``' with your GitHub account name, and
+          '``easybuild-framework``' with the name of the target EasyBuild repository.
+
+i. visit https://github.com/EXAMPLE/easybuild-framework;
+ii. switch to the branch that includes the changes you want to contribute back
+    using the '``Branch: master``' button on the left;
+iii. click the '``New pull request``' button;
+iv. change the target branch to ``develop`` using the '``base: master``' button;
+v. provide an appropriate title and description for your contribution;
+vi. open the pull request by clicking the green '``Create pull request``' button
+
+Next, your pull request will be reviewed & tested, see :ref:`contributing_back_review_process`.
+
+.. _contributing_back_updating_pull_requests:
+
+Updating existing pull requests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note:: This section describes the manual procedure to create a new pull request.
+
+          Please consider using ``eb --update-pr`` instead, see :ref:`github_update_pr`.
+
+It is quite common to update a pull request after creating it, for example
+because Travis reports problems with the changes being made, or as a response
+to someone reviewing your contribution.
+
+To update an existing pull request, it suffices to add commits to the branch
+that was used for opening the pull request, and pushing the updated branch to GitHub.
+
+For example, to update the pull request that was created using the ``mybranch`` branch:
+
+.. code:: shell
+
+  git checkout mybranch
+  # make changes...
+  git commit -am "example commit message for additional changes"
+  git push myfork mybranch
```

Comment:
`git push` is enough in this case. Simplifies things, but might create confusion as to why the first time it wasn't necessary. Up to you if you want to simplify the typing